src/bundle/partial.ts

```diff
@@ -27,49 +43,65 @@ export async function prepareBundlePartialFiles(path: string, snag: LogSnag, org
   return manifest
 }
 
-export async function uploadPartial(apikey: string, manifest: manifestType, path: string, options: Options, appId: string, name: string) {
-  const spinner = spinnerC()
-  spinner.start('Preparing partial update')
-  const uploadResponse: uploadUrlsType[] = await manifestUploadUrls(apikey, appId, name, manifest)
+export async function uploadPartial(apikey: string, manifest: manifestType, path: string, options: any, config: CapacitorConfig, appId: string, name: string) {
+  try {
+    const baseUrl = config?.plugins?.CapacitorUpdater?.cloudflareBaseUrl ?? defaultApiHost
+    const url = new URL(`${baseUrl}/private/partial_upload/v1`)
```

Comment:
Do not put v1 in the URL, we will do that in the headers later

Reply:
You cannot route in hono based on headers. It doesn't hurt to keep the v1